obspy/taup/tau.py

```diff
@@ -663,3 +728,174 @@ def create_taup_model(model_name, output_dir, input_dir):
     else:
         model_file_name = model_name + ".tvel"
     TauPCreate.main(model_file_name, output_dir, input_dir)
+
+
+def traveltime_plot(source_depth, min_degree=10, max_degree=90, nrecs=1000,
+                    phases=['P', 'S'], model='iasp91', fig=None, ax=None):
+        """
+        Returns a travel time plot and any created axis instance of this
+        plot.
+
+        :param source_depth: Source depth in kilometers.
+        :type source_depth: float
+        :param min_degree: minimum distance from the source (in degrees) to a
+            plot travel times Defaults to ``10``.
+        :type min_degree: float
+        :param max_degree: maximum distance from the source (in degrees) to
+            plot travel times. Defaults to ``90``.
+        :type max_degree: float
+        :param nrecs: Number of points to plot. Defaults to ``1000``.
+        :type nrecs: int
+        :param phases: List of phase names which should be used within the
+            plot. Defaults to ``['P', 'PKP']``.
+        :type phases: list of str, optional
+        :param model: string containing the model to use. Defaults to 'iasp91'.
+        :type model: str:
+        :param fig: Figure to plot into. If not given, a new figure instance
+            will be created.
+        :type fig: :class:`matplotlib.axes.Axes
+        :param ax: Axes to plot into. If not given, a new figure with an axes
+            will be created.
+        :type ax: :class:`matplotlib.Figure.figure`
+
+        :returns: ax
+        :rtype: :class:`matplotlib.axes.Axes`
+
+        .. rubric:: Example
+
+        >>> from obspy.taup import traveltime_plot
+        >>> import matplotlib.pyplot as plt
+        >>> fig, ax = plt.subplots()
+        >>> ax = traveltime_plot(source_depth=10, ax=ax, fig=fig)
+        >>> plt.show()
+
+        .. plot::
+
+        from obspy.taup import traveltime_plot
+        import matplotlib.pyplot as plt
+
+        fig, ax = plt.subplots()
+
+        ax = traveltime_plot(source_depth=10, ax=ax, fig=fig)
+
+        plt.show()
+
+        """
+
+        # compute the requested arrivals:
+        model = TauPyModel(model)
+
+        plotted = False  # a flag to check if any arrival exists
+
+        # calculate the arrival times and plot vs. epicentral distance:
+        degrees = np.linspace(min_degree, max_degree, nrecs)
+        for degree in degrees:
+            try:
+                arrivals = model.get_ray_paths(source_depth, degree,
+                                               phase_list=phases)
+                arrivals.timeplot(show=False, ax=ax)
+            except ValueError as err:
+                print(err)
+            else:
+                plotted = True
+
+        if not plotted:
+            raise ValueError("No travel times to plot")
+        return ax
+
+
+def raypath_plot(source_depth, min_degree=10, max_degree=90, nrecs=20,
```

Comment:
Same comments as before. I think `plot_raypaths()` is a better name and the arguments might benefit from different names.